substrate/frame/support/src/traits/schedule.rs

```diff
@@ -130,7 +130,8 @@ impl<T: Decode, H> MaybeHashed<T, H> {
 	}
 }
 
-// TODO: deprecate
+#[allow(deprecated)]
```

Comment:
Do you really need those?  

I think it should rather go on the implementation of any of the traits.

Reply:
I have added a commit to resolve your feedback